packages/user-menu/src/UserNavigation/UserNavigation.js

```diff
@@ -10,16 +10,31 @@ import {
   myEloketUrl,
 } from './UserNavigationUrls';
 
+function renderNotificationsMenuItem(notificationsCount) {
+  if (notificationsCount) {
+    return (
+      <li>
+        <a href={notificationsUrl()} title="Notificaties" alt="Klikt hier om uw notificaties te bekijken.">
+          U heeft notificaties:
```

Comment:
Ik dacht dat Tom V in het Jira ticket daarjuist nog die tekst wou laten wijzigen...

Reply:
yes changed it